docker-compose.yml

```diff
@@ -80,6 +73,23 @@ services:
       - "-Dcom.sun.management.jmxremote.local.only=false"
       - "-Djava.rmi.server.hostname=localhost"
 
+  batch:
+    image: raster-foundry-batch
+    volumes:
+      - ./app-tasks/rf/:/opt/raster-foundry/app-tasks/rf/
+      - ./app-backend/batch/target/scala-2.11/rf-batch.jar:/opt/raster-foundry/jars/rf-batch.jar
+      - $HOME/.aws:/root/.aws:ro
+    build:
+      context: ./app-tasks
+      dockerfile: Dockerfile
+    external_links:
+      - postgres:database.service.rasterfoundry.internal
+      - api-server:rasterfoundry.com
+    env_file: .env
+    environment:
+      - RF_HOST=http://rasterfoundry.com:9000
```

Comment:
I didn't look too closely at this, it's copy-paste of the `airflow-worker` task from `docker-compose.airflow.yml`.

I don't think it can be `localhost` though because inside the container it needs a domain to make requests too -- in this case we're making sure `api-server` => `rasterfoundry.com` a few lines above which is why this works. We could rename this though so it's less confusing.